.github/scripts/select-reviewers.js

```diff
@@ -0,0 +1,156 @@
+const STATS_ISSUE_NUMBER = 19428;
+const MEMBERS = [
+  "B-Step62",
+  "BenWilson2",
+  "daniellok-db",
+  "harupy",
+  "serena-ruan",
+  "TomeHirata",
+  "WeichenXu123",
+];
+
+async function loadStats(github, owner, repo) {
+  try {
+    const issue = await github.rest.issues.get({
+      owner,
+      repo,
+      issue_number: STATS_ISSUE_NUMBER,
+    });
+    const match = issue.data.body.match(/```json\n([\s\S]*?)\n```/);
+    if (match) {
+      return JSON.parse(match[1]);
+    }
+  } catch (err) {
+    console.warn(`Warning: Failed to load stats from issue: ${err.message}`);
+  }
+  return { reviewCounts: {} };
+}
+
+async function saveStats(github, owner, repo, stats) {
+  const body = `This issue tracks reviewer assignment counts for fair distribution.
+
+\`\`\`json
+${JSON.stringify(stats, null, 2)}
+\`\`\`
+`;
+
+  await github.rest.issues.update({
+    owner,
+    repo,
+    issue_number: STATS_ISSUE_NUMBER,
+    body,
+  });
```

Comment:
Using an issue as a storage for easier update/reset/check. We could use an artifact but that's 10x complex.